README.md

```diff
@@ -348,7 +348,7 @@ To learn more about columnar storage, check out the [columnar storage README](ht
 
 ## Documentation
 
-If you’re ready to get started with Citus or want to know more, we recommend reading the [Citus open source documentation](https://docs.citusdata.com/en/stable/). Or, if you are using Citus on Azure, then the [Hyperscale (Citus) documentation](https://docs.microsoft.com/azure/postgresql/hyperscale/) is online and available as part of the Azure Database for PostgreSQL docs.
+If you’re ready to get started with Citus or want to know more, we recommend reading the [Citus open source documentation](https://docs.citusdata.com/en/stable/). Or, if you are using Citus on Azure, then the [Azure Cosmos DB for PostgreSQL](https://learn.microsoft.com/en-us/azure/cosmos-db/postgresql/introduction) is online and available as part of the Azure Cosmos DB for PostgreSQL docs.
```

Comment:
1) remove /en-us locale from the microsoft URL
2) same repetition flagged in earlier comment, since our new name on Azure no longer has 2 parts to it. How about instead: "If you are using Citus on Azure, then reading the [Azure Cosmos DB for PostgreSQL docs](XXX link) is the place to start."